source/user/scf.rst

```diff
@@ -389,6 +389,29 @@ More examples can be found in
 :source:`examples/scf/02-ghf.py`.
 
 
+Scalar relativistic correction
+==========
+Scalar relativistic effects can be applied on the one-body operator through eXact-2-component (X2c) Hamiltonian. 
+X2c Hamiltonian can be invoked by decorating the SCF objects with the :func:`.x2c` method, three other equivalent function names are also listed below::
+
+    mf = scf.RHF(mol).x2c()
+    mf = scf.RHF(mol).x2c1e()
+    mf = scf.RHF(mol).sfx2c()
+    mf = scf.RHF(mol).sfx2c1e()
+
+Note that the :func:`.x2c` method only decorates the one-body operatore of SCF objects.
+It can only account for the mass-velocity effect,
+picture change effect and spin-orbit coupling needs to be accounted for elsewhere.
```

Comment:
while the picture-change effect and spin-orbit coupling need to be accounted for elsewhere

(But one might not even be interested in them at all?)

Reply:
That is probably true, but since pyscf is able to treat spin-orbit coupling by ghf+soc operator, I still think perhaps it can stay here just so people know for sure what the SFX2C is capable of.

Reply:
Yes, but one should not say "needs to be accounted for elsewhere" in cases where there is no need to do that.

It'd be better to formulate this as something like: "the scalar relativistic correction only includes the mass-velocity effect, while the picture change and spin-orbit effects are ignored".